proto/debugpb.proto

```diff
@@ -99,3 +103,22 @@ message ScanMvccResponse {
     bytes key = 1;
     kvrpcpb.MvccInfo info = 2;
 }
+
+message Failure {
+    enum Type {
```

Comment:
Maybe we should figure out a better name.

Reply:
Add two methods instead, one for setting and one for removal.